run

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/python
```

Comment:
How important is this `run` file, and when might I use it?

Reply:
Git acceident, squashed.